### PR TITLE
Update README.md to include links to separate files

### PR DESCRIPTION
**Pull request by NiranjanAP**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ docker pull dhiway/cord:develop
 ```
 
 
-### Create an account and ask for membership.
+### Create an account and ask for membership ( [Account.md](https://github.com/dhiway/confidex/blob/main/Account.md) ).
```
Comment (amarts) [inline README.md:26]: the ideal way is having the relative path inside the repo. That will enable it to rename repo without any problem with links.

Comment (NiranjanAP) [reply README.md:26]: am thinking what'll happen if the files are individually copied over to some other locations

Comment (amarts) [reply README.md:26]: lets keep the link. If the files are renamed or moved to other locations, then that needs to be handled with proper link change here.

Comment (amarts) [inline README.md:26]:
```suggestion
### Create an account and ask for membership ( [Account.md](./Account.md) ).
```

File: README.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ One can install [Polkadot.JS browser extension](https://polkadot.js.org/extensio
 Import the account created from the above into polkadot extension here using the steps mentioned [here](https://docs.cord.network/cord/createaccounts/)
 
 
-#### Generate **'node key'**
+#### Generate **'node key'** ([Node.md](https://github.com/dhiway/confidex/blob/main/Node.md))
```
Comment (amarts) [inline README.md:53]:
```suggestion
#### Generate **'node key'** ([Node.md](./Node.md))
```

File: README.md [hunk 3]
```diff
@@ -63,7 +63,7 @@ In a ledger, each process is not just identified by 'IP:port', but with a specif
 This above command gives the output of public information of nodeKey, something like `12D3KooWMzmZ2g7LBPQuTBsb2HVTszsAToSzXuJJNQzHvbPKWbnq`. This also needs to be stored securely. Also note, similar to key generate, `generate-node-key` too generates a random key for every invocation.
 
 
-### Send the account information (public info) to council
+### Send the account information (public info) to council ([Adding Account And Node to Network.md](https://github.com/dhiway/confidex/blob/main/Adding%20Account%20And%20Node%20to%20Network.md))
```
Comment (amarts) [inline README.md:66]:
```suggestion
### Send the account information (public info) to council ([Adding Account And Node to Network.md](./Adding%20Account%20And%20Node%20to%20Network.md))
```

File: README.md [hunk 4]
```diff
@@ -89,7 +89,7 @@ If anyone wants to add a node, the nodeId (the public part of node.key generated
 After this is voted successfully by the council, one can start the node. Notice that, one account can have only one nodeId. So, if one wants to run 3 nodes, 3 accounts needs to be created.
 
 
-## Starting a data node
+## Starting a data node ([DataNote.md](https://github.com/dhiway/confidex/blob/main/DataNode.md))
```
Comment (amarts) [inline README.md:92]:
```suggestion
## Starting a data node ([DataNote.md](./DataNode.md))
```

File: README.md [hunk 5]
```diff
@@ -123,7 +123,7 @@ Once the logs have messages like below you are successfully started.
 2023-10-29 15:22:58 💤 Idle (8 peers), best: #32155 (0xcc33…bb8a), finalized #32152 (0x1395…62a1), ⬇ 10.2kiB/s ⬆ 5.8kiB/s    
 ```
 
-## Starting a RPC node (used for writes)
+## Starting a RPC node (used for writes) ([RPCNode.md](https://github.com/dhiway/confidex/blob/main/RPCNode.md))
```
Comment (amarts) [inline README.md:126]:
```suggestion
## Starting a RPC node (used for writes) ([RPCNode.md](./RPCNode.md))
```

File: README.md [hunk 6]
```diff
@@ -157,7 +157,7 @@ Once the logs have messages like below you are successfully started.
 2023-10-29 15:22:58 💤 Idle (8 peers), best: #32155 (0xcc33…bb8a), finalized #32152 (0x1395…62a1), ⬇ 10.2kiB/s ⬆ 5.8kiB/s    
 ```
 
-## Adding a new validator into ledger
+## Adding a new validator into ledger ([Validator.md](https://github.com/dhiway/confidex/blob/main/Validator.md))
```
Comment (amarts) [inline README.md:160]:
```suggestion
## Adding a new validator into ledger ([Validator.md](./Validator.md))
```

File: README.md (hunk 1 above)
Comment (NiranjanAP) [reply README.md]: I mean - if the users copy it over to some other locations, then these links will not work

**Pull request merged**